CoreAnalyticView/src/au/gov/asd/tac/constellation/views/analyticview/state/AnalyticViewState.java

```diff
@@ -65,7 +66,11 @@ public void setCurrentAnalyticQuestionIndex(final int currentAnalyticQuestionInd
 
     public void addAnalyticQuestion(final AnalyticQuestionDescription<?> question, final List<SelectableAnalyticPlugin> selectablePlugins) {
         activeAnalyticQuestions.add(question);
-        activeSelectablePlugins.add(selectablePlugins);
+        for (SelectableAnalyticPlugin plugin : selectablePlugins) {
```

Comment:
WHy are you doing this? In fact why is activeSelectablePlugins a list of lists?

Reply:
I had the impression it was a design decision for something that may be implemented in the future possibly to do with saving/loading of questions/plugins.

Reply:
Sorry, I recalled why it was structured this way after I wrote this comment - please disregard.